<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Hp_Laptop" failures="1" tests="2" name="Test1" time="0.091" errors="0" timestamp="2025-07-22T00:35:58 PKT">
  <testcase classname="TestNg.Practice_One" name="Demo" time="0.036">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at TestNg.Practice_One.Demo(Practice_One.java:13)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1575)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- Demo -->
  <testcase classname="TestNg.Practice_One" name="SecondDemo" time="0.002"/>
</testsuite> <!-- Test1 -->
